samples/canopen/canopen_rpdo/RPDOCanNode.hpp

```diff
@@ -0,0 +1,226 @@
+/**
+ * Representation of the CAN node. Handles constructing the object
+ * dictionary and other baseline settings. The idea is that each "board"
+ * will have a specific object dictionary associated with it. The object
+ * dictionary itself will also need to have information on "data of interest".
+ * For example, a temperature management system may to expose water pump
+ * flow rate in the object dictionary.
+ */
+#include <stdint.h>
+
+#include <co_core.h>
+
+class RPDOCanNode {
+public:
+    RPDOCanNode();
+
+    /**
+     * Expose a way to programmatically update the sampleData.
```

Comment:
This function documentation is funky, but I think it should be fine for the purposes of an example